### ISSUE TITLE
pylint_django does not pass flake8

### ISSUE THREAD

**Issue by bittner** [PyCQA/pylint-django#102]
I know, this may sound a bit shady as a comment, but the project does not pass `flake8`. Not even when you ignore line length generously:

```bash
$ flake8 --max-line-length=120
./compat.py:5:5: F401 'astroid.nodes.Class as ClassDef' imported but unused
./compat.py:5:5: F401 'astroid.nodes.Function as FunctionDef' imported but unused
./compat.py:5:5: F401 'astroid.nodes.From as ImportFrom' imported but unused
./compat.py:5:5: F401 'astroid.nodes.AssName as AssignName' imported but unused
./compat.py:5:5: F401 'astroid.nodes.Getattr as Attribute' imported but unused
./compat.py:17:9: F401 'astroid.util.Uninferable' imported but unused
./plugin.py:10:1: F401 'pylint_django.transforms' imported but unused
./transforms/transforms/django_utils_translation.py:1:1: E731 do not assign a lambda expression, use a def
./transforms/transforms/mongoengine.py:4:1: E302 expected 2 blank lines, found 1
./transforms/transforms/mongoengine.py:14:5: E731 do not assign a lambda expression, use a def
./transforms/transforms/django_db_models.py:3:1: E302 expected 2 blank lines, found 1
./transforms/transforms/django_forms.py:14:5: E731 do not assign a lambda expression, use a def
./transforms/transforms/django_forms_fields.py:104:5: E303 too many blank lines (2)
./transforms/transforms/django_db_models_fields_files.py:12:1: W391 blank line at end of file
./augmentations/__init__.py:390:5: E265 block comment should start with '# '
./augmentations/__init__.py:391:5: E265 block comment should start with '# '
./checkers/__init__.py:9:60: W292 no newline at end of file
```

This may not be a big deal, but it's also something that's easily fixed.

Not sure though if you want "another tool" in this repository.

**Comment by bittner** [comment]
Congratulations on the move to PyCQA!

Now it's time to clean up the code base.

**Comment by atodorov** [comment]
@bittner care to contribute a pull request for the flake8 stuff ?

**Comment by bittner** [comment]
I do care.

**Comment by bittner** [comment]
Just a quick summary.

Results of PR #126 and #127:

- We now have made [most parts of the code base](https://travis-ci.org/PyCQA/pylint-django/jobs/338127371#L473) follow PEP8 and other rules enforced by flake8
- We have a build matrix on Travis CI that shows [which checks and tests are failing](https://travis-ci.org/PyCQA/pylint-django) (Python 2.7 mostly for test data, flake8 for code in tests and the `compat` module, Pylint with approx. 80 complaints)
- We're [allowing to fail](https://github.com/PyCQA/pylint-django/blob/master/.travis.yml#L18-L21) the mentioned build jobs in order to get a green build

What we now need is:

1. Someone to look at the **remaining flake8 complaints** (which appear to be dangerous to adjust w/o knowing what the code base does)
1. An explanation / decision whether **Python 2.7 should be supported** again (and someone to look into the issues that appear to be related to the test setup)
1. Probably a separate PR for **fixing Pylint complaints** in the end (if this is considered worth a contribution)

Bullet 1. is required to close this very issue. I'd be happy to assist, though I'm hesitant to do this myself for the reasons pointed out above. (I'm not fluent with the code base, and I don't want to be guilty for breaking the plugin due to this. I'm sure you understand.)

How can we proceed? Any suggestions?
